### PR TITLE
Travis Image update

### PR DESCRIPTION
**Pull request by umeshp7**
Update travis image from trusty to xenial.
Updated ci-requirements for swift, java-okhttp, java-unirest, csharap-restsharp, lib-curl. 
Unskipped newman tests for csharp-restsharp, java-unirest, java-okhttp, shell-httpie, and lib-curl.

### CODE REVIEW

File: codegens/libcurl/testFile.c [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
```
Comment (shreys7) [inline codegens/libcurl/testFile.c]: Do we need to add this file ? @umeshp7

File: codegens/java-okhttp/test/unit/fixtures/testcollection/collection.json [hunk 2]
```diff
@@ -978,373 +978,6 @@
 				"description": "The HTTP `POST` request method is meant to transfer data to a server \n(and elicit a response). What data is returned depends on the implementation\nof the server.\n\nA `POST` request can pass parameters to the server using \"Query String \nParameters\", as well as the Request Body. For example, in the following request,\n\n> POST /hi/there?hand=wave\n>\n> <request-body>\n\nThe parameter \"hand\" has the value \"wave\". The request body can be in multiple\nformats. These formats are defined by the MIME type of the request. The MIME \nType can be set using the ``Content-Type`` HTTP header. The most commonly used \nMIME types are:\n\n* `multipart/form-data`\n* `application/x-www-form-urlencoded`\n* `application/json`\n\nThis endpoint echoes the HTTP headers, request parameters, the contents of\nthe request body and the complete URI requested."
 			},
 			"response": []
-		},
-		{
-			"name": "HEAD request",
```
Comment (shreys7) [inline codegens/java-okhttp/test/unit/fixtures/testcollection/collection.json]: We will need to add unit tests for this deleted requests

Comment (umeshp7) [reply codegens/java-okhttp/test/unit/fixtures/testcollection/collection.json]: Yes, will add unit tests.

File: codegens/libcurl/testFile.c (hunk 1 above)
Comment (umeshp7) [reply codegens/libcurl/testFile.c]: Nope, will delete and include this in gitignore.

**Pull request merged**